charts/pulsar/Chart.yaml

```diff
@@ -18,10 +18,10 @@
 #
 
 apiVersion: v2
-appVersion: "2.9.2"
+appVersion: "2.10.1"
 description: Apache Pulsar Helm chart for Kubernetes
 name: pulsar
-version: 2.9.3
+version: 2.10.1
```

Comment:
Can we decouple this PR from the version bump? I know we have historically had a process to bump the version here and trigger a release, but I think we should make releases align with the official Apache Release process.

Reply:
Since I am working on the new release process, I added a commit to remove this change.